utils/store.go

```diff
@@ -11,7 +11,7 @@
 //    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
-package main
+package utils
```

Comment:
Actually this one can stay in mender. Since interface implementation by a struct is implicit, we do not need to move it anywhere. Though I understand why you may want to move it to utils.